product/domain/productBasics.go

```diff
@@ -665,3 +668,17 @@ func (c *CategoryTeaser) CPath() string {
 	}
 	return c.Parent.CPath() + "/" + c.Code
 }
+
+// GetFirst of the badges, returns nil if there is no first badge
+func (b Badges) GetFirst() *Badge {
```

Comment:
```suggestion
func (b Badges) First() *Badge {
```

For new stuff, we could adhere to effective Go: https://golang.org/doc/effective_go.html#Getters